debian/control

```diff
@@ -121,13 +121,48 @@ Description: Translation files for nymead and plugins - translations
  This package provides the translation files for nymead and for the plugins.
 
 
+Package: libnymea-testlib
+Section: libs
+Architecture: any
+Multi-Arch: same
+Depends: ${shlibs:Depends},
+         ${misc:Depends},
+         libnymea-core (= ${binary:Version})
+Description: Library for nymea tests - library
+ The nymea daemon is a plugin based IoT (Internet of Things) server. The
```

Comment:
I would honestly just drop this whole description :D

It's quite outdated (as if the "powerful" rule engine would be the only feature) and also technically speaking not correct (as a package doesn't install anything, just contains stuff) and also, it's enough if the nymea meta-package has this all-round description. Smaller packages should have something more precise. So if anything:

```
Package: nymea-tests
Description: nymea automated tests
  Automated tests for nymea:core

Package: libnymea-tests
Description: nymea-tests library
  Shared resources for nymea-tests

Package: libnymea-tests-dev
Description: nymea-tests library - development files
  nymea-tests development files for building automated tests.
```

Reply:
This also goes for the libnymea-core package. My proposal would be:

```
Package: libnymea-core
Description: nymea:core internal library
  Internal library for nymea:core

Package: libnymea-core-dev
Description: nymea:core internal library - development files
  Development files for the internal nymea:core library
```